.eslintrc.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     es6: true
   },
   globals: {
-    GOVUK: 'readonly'
+    GOVUK: 'readonly',
+    dataLayer: 'readonly'
```

Comment:
@minhngocd this is to silence the linter on `ga4-page-view-tracking.js` - can also disable it inline 🤔